gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
```

Comment:
@as3810t I think this should update Gradle to 7.4.2. See build logs:
```
$ ./gradlew -v

------------------------------------------------------------
Gradle 7.4.2
------------------------------------------------------------

```

Reply:
You are right, my mistake!

Reply:
Btw I used `./gradlew wrapper --gradle-version 7.4.2` to achieve this